### ISSUE TITLE
Inaccessible "data" property

### ISSUE THREAD

**Issue by schoenwaldnils** [foo-software/lighthouse-check-action#14]
Hi there, I get following error message:

```
##[error]All URLs failed to be enqueued. Examine the "data" property of this error for details.
```

Sadly I can't find a way to access the "data" property.
The job is failing. 
The verbose option isn't getting me any more data than without it.
Can't access `lighthouseCheckResults` since the first step is failing.
Uploading artifacts isn't possible for the same reason.

My config:
```
  lighthouse-check:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@master
      - name: Run Lighthouse
        uses: foo-software/lighthouse-check-action@master
        id: lighthouseCheck
        with:
          accessToken: ${{ secrets.GITHUB_TOKEN }}
          apiToken: ${{ secrets.LIGHTHOUSE_TOKEN }}
          urls: 'https://18.pr.timeline.schoen.world'
          verbose: true
```

Failing action: https://github.com/schoenwaldnils/timeline/pull/18/checks?check_run_id=546516868

Any idea what could help?

**Comment by adamhenson** [comment]
Thanks for reporting this @schoenwaldnils and providing the example. I'll investigate.

**Comment by adamhenson** [comment]
@schoenwaldnils this is now fixed in the latest release. I understand why you're getting an error also, so let me explain. This project can seem a bit complicated and the documentation isn't in an ideal state. The documentation attempts to explain things, but I think it's still not clear to people, understandably.

> This project provides **two ways of running audits** - "locally" by default in a dockerized GitHub environment or remotely via [Automated Lighthouse Check](https://www.automated-lighthouse-check.com) API.

Still confused? I wouldn't be surprised. To run Lighthouse "remotely" we're talking about creating an account on [Automated Lighthouse Check](https://www.automated-lighthouse-check.com) and adding URLs to your account.

The [GitHub docs explain](https://github.com/foo-software/lighthouse-check-action#usage-automated-lighthouse-check-api) how to set things up, but more specifically [how to set things up when you only want to run audits on a subset of URLs](https://github.com/foo-software/lighthouse-check-action#trigger-audits-on-only-certain-pages-in-an-account).

## tl;dr

From the link above the documentation subtly explains that `urls` should be the page tokens in this case. I'm sorry this is not clear. I'm going to open an issue about this.

## Conclusion

You will probably now see this.

```
lighthouse-check: Below is the API response for attempted URLs as an array. [
  {
    code: 'ERROR_PAGE_MISSING',
    message: 'This page does not exist.',
    status: 422
  }
]
```

You will need to update `urls` to use page tokens of your account as [explained here](https://github.com/foo-software/lighthouse-check-action#trigger-audits-on-only-certain-pages-in-an-account).

**Comment by schoenwaldnils** [comment]
Thanks @adamhenson for the investigation.
It seems this renders this action not suitable for the task I intended it for :/
I deploy each pull request on its own subdomain and then would like to run a lighthouse test on it.
With Zeit Now this seems possible. But not with a self build deployment on AWS.

Thanks for the help nonetheless 👍

**Comment by adamhenson** [comment]
No problem @schoenwaldnils. If you have an ephemeral or custom URL, you can use `{page token}::{custom url}` syntax. I provided an [example with ZEIT Now](https://github.com/foo-software/lighthouse-check-action#usage-zeit-now), but you could follow the same example with any custom URL you might have. You would just need a way to expose that URL from whatever steps generates it (using [`output`](https://help.github.com/en/actions/building-actions/metadata-syntax-for-github-actions#outputs)). Not sure if that helps.
